assets/src/apps/v2/on_bus.tsx

```diff
@@ -0,0 +1,50 @@
+import initSentry from "Util/sentry";
+initSentry("on_bus");
+
+import initFullstory from "Util/fullstory";
+initFullstory();
+
+import "../../../css/on_bus_v2.scss";
+
+import React from "react";
+import ReactDOM from "react-dom";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import ScreenPage from "Components/v2/screen_page";
+import { MappingContext } from "Components/v2/widget";
+import MultiScreenPage from "Components/v2/multi_screen_page";
+import SimulationScreenPage from "Components/v2/simulation_screen_page";
+import Placeholder from "Components/v2/placeholder";
+
+const TYPE_TO_COMPONENT = {
+  body: Placeholder,
```

Comment:
I'm a little confused about how this works, if it does. The generated `Placeholder` widget should serialize with a `type` of `placeholder` (see the implementation of `widget_type/1`), which should then appear in this mapping. Instead this is expecting a widget type of `body`, which I don't think any widget has.